btree/btree.c

```diff
@@ -1478,6 +1483,18 @@ static void node_move(struct nd *src, struct nd *tgt,
 }
 #endif
 
+#ifndef __KERNEL__
+static void node_lock(struct nd *node)
```

Comment:
node lock is taken at other places (like node_get, node_put etc.) as well using direct `m0_rwlock_write_lock()` api call. Please change at those places too.

Reply:
ok

Reply:
done.